perception/sandbox/polygrasp/src/polygrasp/segmentation_rpc.py

```diff
@@ -0,0 +1,56 @@
+import logging
+import a0
+import numpy as np
+import signal
+
+from polygrasp import serdes
+
+topic_key = "segmentation"
+
+log = logging.getLogger(__name__)
+
+
+class SegmentationClient:
+    def __init__(self):
+        self.client = a0.RpcClient(topic_key)
+
+    def segment_img(self, rgbd, min_mask_size=2500):
+        bits = serdes.rgbd_to_capnp(rgbd).to_bytes()
+        result_bits = self.client.send_blocking(bits).payload
+        labels = serdes.capnp_to_rgbd(result_bits)
+
+        num_objs = int(labels.max())
+        obj_masked_rgbds = []
+        obj_masks = []
+
+        for obj_i in range(1, num_objs + 1):
+            obj_mask = labels == obj_i
+
+            obj_mask_size = obj_mask.sum()
+            if obj_mask_size < min_mask_size:
+                continue
+            obj_masked_rgbd = rgbd * obj_mask[:, :, None]
+            obj_masked_rgbds.append(obj_masked_rgbd)
+            obj_masks.append(obj_mask)
+
+        return obj_masked_rgbds, obj_masks
+
+
+class SegmentationServer:
+    def _get_segmentations(self, rgbd: np.ndarray):
```

Comment:
assuming this is intended to be a base class - should document that somewhere right?

Reply:
otherwise why not implemented

Reply:
see third_party subdir